src/main/java/com/google/ehub/servlets/LoginServlet.java

```diff
@@ -15,13 +16,23 @@
 import javax.servlet.http.HttpServletResponse;
 
 /**
-* Handles the GET request from the user login page provided by the Users API.
-*/
+ * Handles the GET request from the user login page provided by the Users API.
```

Comment:
nit: update comment since this servlet also handles POST requests now, not just GET requests.

Reply:
OK I updated this comment in last commit.